pkg/sdk/procedures_gen.go

```diff
@@ -247,3 +253,116 @@ type ProcedureDetail struct {
 	Property string
 	Value    string
 }
+
+// CallProcedureOptions is based on https://docs.snowflake.com/en/sql-reference/sql/call.
+type CallProcedureOptions struct {
+	call              bool                   `ddl:"static" sql:"CALL"`
+	name              SchemaObjectIdentifier `ddl:"identifier"`
+	CallArguments     []string               `ddl:"keyword,must_parentheses"`
+	ScriptingVariable *string                `ddl:"parameter,no_quotes,no_equals" sql:"INTO"`
+}
+
+// CreateAndCallForJavaProcedureOptions is based on https://docs.snowflake.com/en/sql-reference/sql/call-with#java-and-scala.
+type CreateAndCallForJavaProcedureOptions struct {
```

Comment:
Looking at the CreateAndCallForX I started to wander: these are all really similar to the createForX. Should we consider merging/reusing them? I think it would be beneficial but also complicates them... I would just like to have a short discussion about this, it's not a blocking comment, eventually some TODO for the future.

Reply:
thats a good point. i think in this case the names are clear enough to make it easy to understand, and being easy to understand is more important than having shorter code. there could be a way to merge them, but I am not sure it is worth the trouble.

Reply:
LEt's leave it for the future then